### ISSUE TITLE
"Goto definition" stopped working (3.0.0b2 Mac)

### ISSUE THREAD

**Issue by ShenggaoZhu** [spyder-ide/spyder#3013]
## Description

**What steps will reproduce the problem?**

1. Start Spyder
2. Hold "Command" key on Mac, and click on the function/module name

**What is the expected output? What do you see instead?**

The editor should jump to the definition, but it didn't...

**Please provide any additional information below**

"Goto definition" worked before. I recently updated my anaconda, and I think that update may break it...

## Version and main components

* Spyder Version: 3.0.0b2 
* Python Version: 2.7.11
* Qt Versions: 5.4.1, PyQt5 5.4.1 on Darwin

## Dependencies
```
IPython >=3.0   :  4.0.3 (OK)
matplotlib >=1.0:  1.5.1 (OK)
numpy >=1.7     :  1.10.4 (OK)
pandas >=0.13.1 :  0.17.1 (OK)
pep8 >=0.6      :  1.7.0 (OK)
psutil >=0.3    :  3.4.2 (OK)
pyflakes >=0.5.0:  1.0.0 (OK)
pygments >=1.6  :  2.1 (OK)
pylint >=0.25   :  1.5.2 (OK)
qtconsole >=4.0 :  4.1.1 (OK)
rope >=0.9.4    :  0.9.4-1 (OK)
sphinx >=0.6.6  :  1.3.3 (OK)
sympy >=0.7.3   :  0.7.6.1 (OK)
zmq >=13.0.0    :  15.2.0 (OK)

```

**Comment by ShenggaoZhu** [comment]
Also, code completion stopped working in the Editor, but it still works in the IPython console.

**Comment by ccordoba12** [comment]
This should be fixed by pull request #2932. Please reopen this bug if that's not the case after we release beta3.

**Comment by ShenggaoZhu** [comment]
Yes, I just pulled the latest dev version, and go-to-definition and code completion are working perfectly now! Thanks!
